packages/react-three-rapier/src/AnyCollider.tsx

```diff
@@ -131,6 +131,7 @@ export const AnyCollider = memo(
           quaternion={quaternion}
           scale={scale}
           ref={ref}
+          name={name}
```

Comment:
I didn't need to add the same thing to RidigBody since it uses the spread operator `(...objectProps)`.